collect_app/src/main/java/org/odk/collect/android/activities/FormChooserList.java

```diff
@@ -88,24 +83,17 @@ public void onCreate(Bundle savedInstanceState) {
                 getString(R.string.sort_by_name_asc), getString(R.string.sort_by_name_desc),
                 getString(R.string.sort_by_date_asc), getString(R.string.sort_by_date_desc),
         };
-    }
 
+        setupAdapter();
```

Comment:
At some point we should rename all these to `setUpAdapter`, since “set up”, as used here, is a phrasal verb.